.gitlab/validate-logs-intgs/validate_log_intgs.py

```diff
@@ -19,6 +19,7 @@
 
 EXCEPTIONS = {
     'amazon_eks': [ERR_UNEXPECTED_LOG_COLLECTION_CAT], # eks is just a tile
+    'aspdotnet': [ERR_MISSING_LOG_DOC], # Use iis pipeline
```

Comment:
Shouldn't we document iis source in the aspdotnet readme?

Reply:
Since it's the exact same example, I think it's better to keep the doc in the iis readme and avoid having out of sync documentation about log collection in the future

Reply:
That's fair, at the same time:
1. It's good that readers don't have to switch to another page, also that they might get confused and change the source.
2. The goal of this log validation is to prevent (some) ouf of sync docs.

I'll let you take the decision, both options are possible